packages/paste-website/src/pages/components/badge/index.mdx

```diff
@@ -279,6 +291,28 @@ but would benefit from the visual affordance of differently-colored Badges. Icon
 `.trim()}
 </LivePreview>
 
+### Brand
+
+Use the brand Badge for cross-selling opportunities across platforms (e.g., Segment, SendGrid, Communications, Flex) and pricing plans. Use these sparingly.
+
+<LivePreview scope={{Box, Badge, ProductEmailAPIIcon, ProductSegmentIcon, ProductCommsIcon, FaxCapableIcon}} language="jsx">
+  {`<Box display="flex" columnGap="space80">
+  <Badge as="span" variant="brand10">
+    <ProductCommsIcon decorative />
+    Twilio CPaaS
+  </Badge>
+  <Badge as="span" variant="brand20">
+    <ProductEmailAPIIcon decorative />
+    Twilio Segment CDP
+  </Badge>
+  <Badge as="span" variant="brand30">
+    <ProductSegmentIcon decorative />
+    Twilio SendGrid
```

Comment:
chore: Use ProductSendGridIcon here (instead of EmailAPI) and swap the text labels